### PR TITLE
launcher-aiming-control

### PR DESCRIPTION
**Pull request by cjdrake42**

### CODE REVIEW

File: Documentation/Signoffs/launcher-aiming-control.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Launcher Aiming Control Unit
+## Functionality
+The launch aiming control unit handles the different functions within the launcher. This includes controlling the signals for the motors for the launcher aiming subsystem, sending a signal for the buzzer to indicate that the launcher is about to fire, and sending a signal to the launcher firing subsystem indicating to fire.
+## Constraints
+C1: The launch aiming control unit shall have the sufficient amount of pins to control the each subsystem
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/launcher-aiming-control.md]: How many is that?

**Review by JesseTNRoberts — COMMENTED** [review]
The constraints for this system have not been thought through. There's no way to design the system unless you have a clear understanding of what it must do.

**Review by JesseTNRoberts — COMMENTED** [review]
This looks like a signoff for the raspberry pi and controls to the launcher. But it's a signoff for the control of the aiming? 

This is a good signoff and I would be willing to approve but I'm confused at what I'm agreeing to.

**Comment by cjdrake42** [comment]
I believe that initially in our conceptual design the launcher aiming system's responsibility was to take in sensor inputs, move the launcher to position, calculate time to fire for the firing system, and send the signal to the firing system. When approaching the detailed design, it was determined that it would make more sense to split the subsystem's parts into two signoffs: the launcher aiming system which highlights the motors needed to move the launcher and the launcher aiming control which provides the control for all of the tasks in the launcher aiming system. I agree that the terminology for the launcher aiming control is not clear because it is performing tasks outside the scope of the launcher aiming system. As a result I have changed the terminology of the system from launcher aiming control to launcher control in the signoff markdown file.

**Comment by micahr234** [comment]
Your block diagram shows that the time to fire is calculated AFTER the launcher is moved to the detection path. But the t_fire (time to fire) variable in your equations is dependent on the t_aim (time to aim). Seems like you will have already aimed when you calculate t_fire. Can you explain?

**Comment by cjdrake42** [comment]
Added a clarification for what t_aim and t_fire is and how they interact with each other

**Review by micahr234 — APPROVED** [review]

**Pull request merged**